executor/include/execution/executor.hpp

```diff
@@ -17,10 +17,10 @@ TYPE_TO_STRING(omp_executor<oneway_t, bulk_t, blocking_t::always_t>);
 TYPE_TO_STRING(cuda_executor<oneway_t, bulk_t, blocking_t::always_t>);
 
 // List of Executor types to run tests for
-#define TEST_EXECUTORS                                       \
-  inline_executor<oneway_t, single_t, blocking_t::always_t>, \
-      sse_executor<oneway_t, bulk_t, blocking_t::always_t>,  \
-      omp_executor<oneway_t, bulk_t, blocking_t::always_t>
+#define TEST_EXECUTORS                                             \
+  const inline_executor<oneway_t, single_t, blocking_t::always_t>, \
+      const sse_executor<oneway_t, bulk_t, blocking_t::always_t>,  \
+      const omp_executor<oneway_t, bulk_t, blocking_t::always_t>
```

Comment:
what's the point of defining these as const if later you're going to cast away the const?

Reply:
Casts are no longer needed. The blocker was the`require` function. Since that's const now, so no need to remove constness now.